tests/testCoordinates.py

```diff
@@ -40,21 +40,25 @@
 import math
 import operator
 
-import lsst.utils.tests as utilsTests
+import lsst.utils.tests
 import lsst.afw.geom as geom
 
-numpy.random.seed(1)
-
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
 
-class CoordinateTestCase(unittest.TestCase):
+class CoordinateTestCase(object):
+    """Mixin for some of the tests below.
+    """
 
-    def assertClose(self, a, b, msg=None):
+    def _assertClose(self, a, b, msg=None):
```

Comment:
I have no idea why it is required. @pschella made the changes here. I would try replacing all of the calls with assertNearlyEqual and kill this method.